<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StillPublishUI</class>
 <widget class="QMainWindow" name="StillPublishUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1081</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QGridLayout" name="gridLayout_7" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="1,1,3">
      <item row="11" column="0" colspan="2">
       <widget class="QPushButton" name="publish_pushButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Publish</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" rowspan="4">
       <layout class="QVBoxLayout" name="snap_verticalLayout" stretch="0">
        <item>
         <widget class="QListWidget" name="publish_listWidget"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="design_radioButton">
          <property name="text">
           <string>Design</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="asset_radioButton">
          <property name="text">
           <string>Asset</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="scene_radioButton">
          <property name="text">
           <string>Scene</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QGridLayout" name="gridLayout" columnstretch="0">
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Status</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Task</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>User</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Source : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="source_lineEdit"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="sourcePublish_horizontalLayout"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="project_layout">
        <item>
         <widget class="QLabel" name="mode_label">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="department_horizontalLayout">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Department : </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="projectPlc_frame">
          <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,0">
           <item row="2" column="1">
            <widget class="QComboBox" name="entitySub2_comboBox"/>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="project_comboBox"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="entitySub1_label">
             <property name="text">
              <string>Type</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="entitySub2_label">
             <property name="text">
              <string>SubType</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Project</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="entitySub1_comboBox"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="entity_label">
             <property name="text">
              <string>Asset</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="entity_comboBox"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="11" column="2">
       <widget class="QLabel" name="status_label">
        <property name="text">
         <string>----</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="9">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="preview_label">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1" rowspan="9">
       <layout class="QVBoxLayout" name="publish_verticalLayout"/>
      </item>
      <item row="0" column="2">
       <layout class="QHBoxLayout" name="header_horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Publish to : </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
